### PR TITLE
Private Offer Updates.md

### PR DESCRIPTION
**Pull request by marcellarichmond**
## Status
In Progress

## Related Issues
fixes: link to the issue

## Description
Updating several pages for private offer launch. 

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by marcellarichmond** [comment]
Not ready to be reviewed and listed yet. More changes to be made.

File: docs/partners/private-offer.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+![NewPrivateOfferPartnerSalesProcessFlow](../doc_imgs/partners/NewPrivateOfferPartnerSalesProcessFlow.png)
+
+<img src="../doc_imgs/partners/NewPrivateOfferPartnerSalesProcessFlow.png" width="200px;"/>
+
+## Private Offer Listing Designation 
+Let customers know your XSOAR Marketplace Premium content pack is available for Private Offer! For partners who elect to participate in Private Offer, the UI will be updated to advertise and drive attention to your pack listing as shown below:
+
+![PrivateOffer-greenarrow](../doc_imgs/partners/PrivateOffer-greenarrow.png)
+
+For Partners interested in having their premium content available for Private Offer on the XSOAR Marketplace, please contact the XSOAR Partner Success Team (PST) at __xsoar.privateoffer@paloaltonetworks.com__ to ensure your solution is added to our internal enablement content.
```
Comment (mchasepan) [inline docs/partners/private-offer.md:44]: fix link

File: docs/partners/private-offer.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+Let customers know your XSOAR Marketplace Premium content pack is available for Private Offer! For partners who elect to participate in Private Offer, the UI will be updated to advertise and drive attention to your pack listing as shown below:
+
+![PrivateOffer-greenarrow](../doc_imgs/partners/PrivateOffer-greenarrow.png)
+
+For Partners interested in having their premium content available for Private Offer on the XSOAR Marketplace, please contact the XSOAR Partner Success Team (PST) at __xsoar.privateoffer@paloaltonetworks.com__ to ensure your solution is added to our internal enablement content.  
+
+To create your Private Offer listing, follow the [Premium Packs setup](https://xsoar.pan.dev/docs/packs/premium_packs) and then follow these steps to add the Private Offer button:
+
+<details>
+  <summary>Click here for detailed steps to add Private Offer to a Content Pack</summary>
```
Comment (mchasepan) [inline docs/partners/private-offer.md:49]: @ShahafBenYakir this doenst look right in the preview

File: docs/partners/private-offer.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+1. In your pack directory to your [Pack Readme](https://xsoar.pan.dev/docs/documentation/pack-docs#pack-readme) (`Packs/MY_PACK/README.md`).
+2. In top of the README file copy and paste the following Markdown text:
+```markdown
+[![image](https://raw.githubusercontent.com/demisto/content/master/Images/request_private_offer_button.png)](https://xsoar.pan.dev/request-private-offer)
+[Learn about Private Offer](https://xsoar.pan.dev/private-offer-learn-more)
+```
+**Make sure to copy the text exactly as it is.**
+</details>
+
+Contact __xsoar.privateoffer@paloaltonetworks.com__ for help.
```
Comment (mchasepan) [inline docs/partners/private-offer.md:60]: fix link

File: docs/partners/private-offer.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+</details>
+
+Contact __xsoar.privateoffer@paloaltonetworks.com__ for help. 
+
+## Working with the Palo Alto Networks Sellers
+Whether the Private Offer is initiated by the customer or the Partner sales rep, alignment with the Cortex seller during the Private Offer process will be instrumental in ensuring a seamless and successful customer sales experience.
+
+The Cortex seller will be the main point of contact and lead the customer through the sales cycle and Private Offer process. To ensure the customer’s requirements are being met with the proposed solution, the Cortex seller will engage with the Partner sales team as needed for technical validation, demos, etc. They will also work closely with the Partner business team for guidance and approval on the requested offer terms. 
+
+**Have a Private Offer Opportunity?** Please email the details of the request to: __xsoar.privateoffer@paloaltoneworks.com__.
```
Comment (mchasepan) [inline docs/partners/private-offer.md:67]: fix link

File: docs/partners/private-offer.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+## Private Offer Agreement 
+The Agreement details the terms and conditions for delivering Partner content to a specific Customer through the XSOAR Marketplace. The Agreement represents a binding contract between the Partner and Palo Alto Networks [(see sample here)](../doc_imgs/partners/PrivateOfferTermsAgreementSample.pdf).
```
Comment (mchasepan) [inline docs/partners/private-offer.md:70]:
```suggestion
The Agreement details the terms and conditions for delivering Partner content to a specific Customer through the XSOAR Marketplace. The Agreement represents a binding contract between the Partner and Palo Alto Networks ([see sample](../doc_imgs/partners/PrivateOfferTermsAgreementSample.pdf)).
```

File: docs/partners/private-offer.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+## Enabling Cortex Sales to sell Partner Solutions
+We are invested in the success of the XSOAR Marketplace and want to help maximize our Partner’s efforts and investment in developing premium content available for Private Offer. 
+
+On an ongoing basis, we will be running internal enablement sessions to ensure sales understands the value Partner solutions add to a customer’s XSOAR investment. To aid in enabling our sellers to promote and sell your solution we recommend creating the following:
+* [Sales playbook](https://docs.google.com/presentation/d/1WMrFTYoN-wc8BEwFnTnetNG4-lPP3wAiD8g-W0gSMbI/edit?usp=sharing) (e.g. cheat sheet) to highlight Partner solution, value prop and use cases.
+* [Joint Solution Brief](https://docs.google.com/document/d/1Hk2fCSHd6dKAQo0eEUyBx8y1oSK4j-17AcaV-pQRnf4/edit?usp=sharing) (JSB)
+* Please send completed playbooks and JSBs to the XSOAR PST for review: __xsoar.privateoffer@paloaltonetworks.com__.
+
+## Ways to promote your Private Offer solution(s)
+* Write a blog post [template here](https://docs.google.com/document/d/1amTRiQ2fl_wUNvfF8M4MP939PlK2x-nyCQLqzjsV_eg/edit?usp=sharing)
```
Comment (mchasepan) [inline docs/partners/private-offer.md:83]:
```suggestion
* Write a blog post [template here](https://docs.google.com/document/d/1amTRiQ2fl_wUNvfF8M4MP939PlK2x-nyCQLqzjsV_eg/edit?usp=sharing) (must adhere to Palo Alto Networks branding and logo [usage guidelines](https://www.paloaltonetworks.com/company/brand.html))
```

File: docs/partners/private-offer.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+* Enable your sales teams
+* Include Cortex XSOAR in your materials
+* Tweet about the offering
+
+*must adhere to Palo Alto Networks branding and logo [usage guidelines](https://www.paloaltonetworks.com/company/brand.html).
```
Comment (mchasepan) [inline docs/partners/private-offer.md:88]:
```suggestion
```

File: docs/partners/private-offer.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+## Still have questions? 
+For more information about Private Offer, please review our [Partner FAQ](https://docs.google.com/document/d/1kUFQPKmuiJuFHNtrg6RHg4cEMeAkjGsDNPXK9Etf4eY/edit#heading=h.pe5n8bsuhdc)(restricted access). For questions or help, please contact the XSOAR PST at __xsoar.privateoffer@paloaltonetworks.com__.
```
Comment (mchasepan) [inline docs/partners/private-offer.md:91]:
```suggestion
For more information about Private Offer, please review our [Partner FAQ](https://docs.google.com/document/d/1kUFQPKmuiJuFHNtrg6RHg4cEMeAkjGsDNPXK9Etf4eY/edit#heading=h.pe5n8bsuhdc) (restricted access). For questions or help, please contact the [Partner Success Team](mailto:xsoar.privateoffer@paloaltonetworks.com). 
```

**Review by mchasepan — COMMENTED** [review]
Looking GREAT @marcellarichmond !!!!

File: docs/partners/private-offer.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+
+On an ongoing basis, we will be running internal enablement sessions to ensure sales understands the value Partner solutions add to a customer’s XSOAR investment. To aid in enabling our sellers to promote and sell your solution we recommend creating the following:
+* [Sales playbook](https://docs.google.com/presentation/d/1WMrFTYoN-wc8BEwFnTnetNG4-lPP3wAiD8g-W0gSMbI/edit?usp=sharing) (e.g. cheat sheet) to highlight Partner solution, value prop and use cases.
+* [Joint Solution Brief](https://docs.google.com/document/d/1Hk2fCSHd6dKAQo0eEUyBx8y1oSK4j-17AcaV-pQRnf4/edit?usp=sharing) (JSB)
+* Please send completed playbooks and JSBs to the XSOAR PST for review: xsoar.privateoffer@paloaltonetworks.com.
+
+## Ways to Promote Your Private Offer Solution(s)
+* Write a blog post [(template here)](https://docs.google.com/document/d/1amTRiQ2fl_wUNvfF8M4MP939PlK2x-nyCQLqzjsV_eg/edit?usp=sharing)
+* Enable your sales teams
+* Include Cortex XSOAR in your materials
```
Comment (MMCrosby) [inline docs/partners/private-offer.md:85]:
```suggestion
* Highlight the value prop of your Cortex XSOAR solution and partnership on your website
```

File: docs/partners/private-offer.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+* Tweet about the offering
+
```
Comment (MMCrosby) [inline docs/partners/private-offer.md:87]:
```suggestion
*  Get the word out to your customers (see email template)
* Do a webinar
* Post about it on LinkedIn

```

**Review by MMCrosby — APPROVED** [review]

File: docs/partners/private-offer.md [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+
+Contact xsoar.privateoffer@paloaltonetworks.com for help. 
+
+## Working with the Palo Alto Networks Sellers
+Whether the Private Offer is initiated by the customer or the Partner sales rep, alignment with the Cortex seller during the Private Offer process will be instrumental in ensuring a seamless and successful customer sales experience.
+
+The Cortex seller will be the main point of contact and lead the customer through the sales cycle and Private Offer process. To ensure the customer’s requirements are being met with the proposed solution, the Cortex seller will engage with the Partner sales team as needed for technical validation, demos, etc. They will also work closely with the Partner business team for guidance and approval on the requested offer terms. 
+
+**Have a Private Offer Opportunity?** Please email the details of the request to xsoar.privateoffer@paloaltoneworks.com.
```
Comment (mchasepan) [inline docs/partners/private-offer.md:67]:
```suggestion
**Have a Private Offer Opportunity?** Please email the details of the request to xsoar.privateoffer@paloaltonetworks.com.
```

File: docs/partners/private-offer.md [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+To create your Private Offer listing, follow the [Premium Packs setup](https://xsoar.pan.dev/docs/packs/premium_packs) and then follow these steps to add the Private Offer button:
+
+<details>
+<summary>Click here for detailed steps to add Private Offer to a Content Pack</summary>
+
+1. In your pack directory navigate to your [Pack Readme](https://xsoar.pan.dev/docs/documentation/pack-docs#pack-readme) (`Packs/My_Pack/README.md`).
+2. In top of the README file copy and paste the following Markdown text:
+```markdown
+[![image](https://raw.githubusercontent.com/demisto/content/master/Images/request_private_offer_button.png)](https://xsoar.pan.dev/request-private-offer)
+[Learn about private offer](https://xsoar.pan.dev/private-offer-learn-more)
```
Comment (mchasepan) [inline docs/partners/private-offer.md:54]:
```suggestion
[Learn about Private Offer](https://xsoar.pan.dev/private-offer-learn-more)
```

File: docs/partners/premium-packs-process.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+## 4. Build Premium Pack(s)
+
+Build the content for the premium packs. Customers can subscribe to Premium Packs on a monthly basis or through [Private Offer](https://xsoar.pan.dev/private-offer). 
+
+- If you're submitting a premium content pack, add the dashboards, playbooks, etc. as you would normally. Revisit the [Contributor Guidlines](https://xsoar.pan.dev/docs/contributing/contributing#contributor-guidelines) if needed.
+- If you're submitting a premium resell pack, the pack is mostly empty except for some basic metadata files.
+
+Either way, we'll work with you to define, scope and guide you through the process.
+
+To update your Premium Pack to be available through Private Offer, please follow these [steps](https://xsoar.pan.dev/request-private-offer).
```
Comment (MMCrosby) [inline docs/partners/premium-packs-process.md:29]:
```suggestion
To update your Premium Pack to be available through Private Offer, please follow these [steps](https://xsoar.pan.dev/docs/partners/private-offer#private-offer-listing-designation-steps).

```

**Review by MMCrosby — CHANGES_REQUESTED** [review]

File: docs/partners/premium-packs-process.md [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+
+In order for us to pay the revenue share, publishers of premium packs must complete Palo Alto Networks vendor onboarding. This process is straightforward and begins by completing a form and then you can expect communications with our Onboarding team via email and our secure portal. 
+
+## 3. Create Private Repository
+
+Premium packs are not stored in the XSOAR open source repository but instead in the publisher's own, private GitHub repo. We'll help you [set it up](/docs/packs/premium_packs). 
+
+## 4. Build Premium Pack(s)
+
+Build the content for the premium packs. Customers can subscribe to Premium Packs on a monthly basis or through [Private Offer](https://xsoar.pan.dev/private-offer).
```
Comment (MMCrosby) [inline docs/partners/premium-packs-process.md:22]:
```suggestion
Build the content for the premium packs. Customers can subscribe to Premium Packs on a monthly basis or through [Private Offer](https://xsoar.pan.dev/docs/partners/private-offer). 

```

**Review by MMCrosby — APPROVED** [review]

File: docs/partners/marketplace.md [hunk 15]
```diff
@@ -1,55 +1,71 @@
...
 
-Additionally, Partners have the ability to monetize their content on the Marketplace. You decide the content, the price, and how to best market it, and get a healthy share of the revenue. The customer gets premium content to solve their use case. It's a win-win. Partners who wish to contribute paid packs must follow a few [additional onboarding steps](/docs/partners/paid-packs-process). 
+Additionally, Partners have the ability to monetize their content on the Marketplace. You decide the content, the price, and how to best market it, and get a healthy share of the revenue. The customer gets premium content to solve their use case. It's a win-win. Partners who wish to contribute premium packs must follow a few [additional onboarding steps](https://xsoar.pan.dev/docs/partners/premium-packs-process) and can now take advantage of [Private Offers](https://xsoar.pan.dev/docs/partners/private-offer)!
```
Comment (MMCrosby) [inline docs/partners/marketplace.md:10]:
```suggestion
Additionally, Partners have the ability to monetize their content on the Marketplace. You decide the content, the price, how to best market it, and get a healthy share of the revenue. The customer gets premium content to solve their use case. It's a win-win. Partners who wish to contribute premium packs must follow a few [additional onboarding steps](https://xsoar.pan.dev/docs/partners/premium-packs-process) and can now take advantage of [Private Offers](https://xsoar.pan.dev/docs/partners/private-offer)! 
```

File: docs/partners/marketplace.md [hunk 16]
```diff
@@ -1,55 +1,71 @@
...
 
 ## Packs
 
-All content on Marketplace is included in a Pack. ([What can be in a pack?](/docs/packs/packs-format#directories)) When Partners create a pack, they specify the content that is included in that pack. 
+All content on Marketplace is included in a Pack. [(What can be in a pack?)](https://xsoar.pan.dev/docs/packs/packs-format#directories) When Partners create a pack, they specify the content that is included in that pack.
 
-Packs on Marketplace can be free or paid. Our platform does full dependency checking, too, so a pack can be dependent on, say, a command in another pack. This allows you to build a playbook addressing a full use case across products. Paid Packs are held in your private repository. Free Packs are held in our open source repository. 
+Packs on Marketplace can be free or premium. Our platform does full dependency checking too, so a pack can be dependent on, say, a command in another pack. This allows you to build a playbook addressing a full use case across products. Premium Packs are held in your private repository. Free Packs are held in our open source repository.
```
Comment (MMCrosby) [inline docs/partners/marketplace.md:16]:
```suggestion
Packs on Marketplace can be free or premium. Our platform does full dependency checking too, so a pack can be dependent on, say, a command in another pack. This allows you to build a playbook addressing a full use case across products. Premium packs are held in your private repository. Free packs are held in our open source repository.
```

File: docs/partners/marketplace.md [hunk 17]
```diff
@@ -1,55 +1,71 @@
...
+**Monthly** - Customer subscribes to your premium pack and the subscription automatically renews on a monthly basis.  We automatically pay you a revenue share, also on a monthly basis, with no invoice required!
+**Private Offer:** Customer subscribes to your premium content for a mutually agreed upon term length and price. We automatically pay you a revenue share, up front, for the full contracted term!  
 
-Marketplace provides robust reporting to Partners on their paid pack activity including both:
+Learn more about [Private Offer](https://xsoar.pan.dev/docs/partners/private-offer)
 
-1. **Notifications** are sent immediately via email whenever there is activity on the Pack including a new trial, trial end, a new subscription, etc. Each notification will include customer/human information when possible.
-1. **Reports** are sent on the first of each month where there was revenue-related activity in the prior month. The report details dollar roll-ups per pack, each revenue related transaction line item, etc. 
+For packs that resell 3rd party products, the following illustrates the flow of subscription:
```
Comment (MMCrosby) [inline docs/partners/marketplace.md:39]:
```suggestion
For packs that resell 3rd party products, the following illustrates the subscription flow:
```

**Review by MMCrosby — APPROVED** [review]

File: docs/partners/private-offer.md [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+  * Sell into Enterprise SOC accounts 
+  * Qualified leads for solution portfolio expansion
+* Amplify reach with brand alignment and marketing activities, subject to Palo Alto Networks' discretion
+
+## How Does Private Offer Work? 
+To ensure as seamless an experience as possible for both you and our customers, we have created a simplified process for Private Offer. XSOAR Customers have the ability to request a Private Offer directly within the XSOAR Marketplace user interface by clicking on ![PrivateOfferButton](../doc_imgs/partners/PrivateOfferButton.png) button in participating premium pack listings. Customers then complete a simple [form](https://xsoar.pan.dev/request-private-offer) to submit their Private Offer request. If you choose to participate in Private Offer, you will just need to add the button to your pack listing. 
+
+Requests can also be initiated through your sales teams or our Cortex sales teams. We highly recommend that your teams promote your XSOAR packs to ensure customers/prospects know you have a premium offering in the XSOAR Marketplace. If an opportunity is identified, email the details of the request to xsoar.privateoffer@paloaltonetworks.com.
```
Comment (MMCrosby) [inline docs/partners/private-offer.md:32]:
```suggestion
Requests can also be initiated through your sales teams or our Cortex sales teams. We highly recommend that your teams promote your XSOAR packs to ensure customers/prospects know you have a premium offering in the XSOAR Marketplace. If an opportunity is identified, email the details of the request to xsoar.privateoffer@paloaltonetworks.com. 
```

**Review by MMCrosby — APPROVED** [review]

File: docs/partners/marketplace.md [hunk 19]
```diff
@@ -1,55 +1,71 @@
...
-For packs that resell 3rd party products, the following illustrates the flow of subscription:
+### Subscribing to Premium Packs
 
-![paid pack flow](../doc_imgs/partners/PaidPackFlow.png)
+The Marketplace transacts in points, rather than dollars, such that one point is valued at US$100. The Partner sets the price of the pack in points per month and customers purchase points via their Palo Alto Networks sales channel to subscribe to your content. 
 
-### Reporting and Notifications
+The XSOAR Marketplace supports two subscription models to give customers purchasing flexibility and you more options to help them address new use cases by integrating key SecOps solutions through their XSOAR deployment: 
+- **Monthly** - Customer subscribes to your premium pack and the subscription automatically renews on a monthly basis.  We automatically pay you a revenue share, also on a monthly basis, with no invoice required!
+ -**Private Offer:** Customer subscribes to your premium content for a mutually agreed upon term length and price. We automatically pay you a revenue share, up front, for the full contracted term!
```
Comment (MMCrosby) [inline docs/partners/marketplace.md:35]:
```suggestion
- **Private Offer:** Customer subscribes to your premium content for a mutually agreed upon term length and price. We automatically pay you a revenue share, up front, for the full contracted term!  
```

**Review by MMCrosby — APPROVED** [review]

**Pull request merged**